src/_pytest/logging.py

```diff
@@ -431,13 +436,48 @@ def clear(self) -> None:
         """Reset the list of log records and the captured log text."""
         self.handler.reset()
 
+    def force_enable_logging(
+        self, level: Union[int, str], logger_obj: logging.Logger
+    ) -> int:
+        """Enable the desired logging level if the level was disabled.
+
+        Only enables logging levels equal too and higher than the requested ``level``.
+
+        Does nothing if the desired ``level`` wasn't disabled.
+
+        :param Union[int, str] level:
+            The logger level caplog should capture.
+            All logging is enabled if a non-standard logging level string is supplied.
+            Valid level strings are in :data:`logging._nameToLevel`.
+        :param Logger logger_obj: The logger object to check.
+
+        :return int: The original disabled logging level.
+        """
+        original_disable_level: int = logger_obj.manager.disable  # type: ignore[attr-defined]
+
+        if isinstance(level, str):
+            level = logging.getLevelName(level)
+
+        if not isinstance(level, int):
+            logging.disable(logging.NOTSET)
+            return original_disable_level
+        if logger_obj.isEnabledFor(level):
+            return original_disable_level
+
+        disable_level = max(level - 10, logging.NOTSET)
+        logging.disable(disable_level)
+
+        return original_disable_level
+
     def set_level(self, level: Union[int, str], logger: Optional[str] = None) -> None:
         """Set the level of a logger for the duration of a test.
 
         .. versionchanged:: 3.4
             The levels of the loggers changed by this function will be
             restored to their initial values at the end of the test.
 
+        Will enable the requesed logging level if it was disabled via :meth:`logging.disable`.
```

Comment:
```suggestion
        Will enable the requested logging level if it was disabled via :meth:`logging.disable`.
```

Reply:
I guess my "t" finger was tired that day.

Reply:
fixed